src/target/lpc43xx.c

```diff
@@ -80,40 +80,40 @@ bool lpc43xx_probe(target_s *t)
 	chipid = target_mem_read32(t, LPC43XX_CHIPID);
 
 	switch (chipid) {
-	case 0x4906002B: /* Parts with on-chip flash */
-	case 0x7906002B: /* LM43S?? - Undocumented? */
-		switch (t->cpuid & 0xFF00FFF0) {
-		case 0x4100C240:
+	case 0x4906002b: /* Parts with on-chip flash */
```

Comment:
We would say "Add `U`" here, but that fells unnecessary when v1.10 will see this code replaced wholesale when we land feature/lpc4370-external-flash-support as we rewrote such a large part of this file.